src/dodal/beamlines/b01_1.py

```diff
@@ -50,11 +55,25 @@ def panda() -> HDFPanda:
 
 @device_factory()
 def synchrotron() -> Synchrotron:
+    """The synchrotron light source.
+
+    Returns:
+        Synchrotron: The synchrotron light source device.
+    """
```

Comment:
Could: Arguably not worth bothering with this one :)

Reply:
Yes, it's there for completeness rather than being in anyway helpful!